<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pageoeuvre</class>
 <widget class="QDialog" name="Pageoeuvre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1546</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>Modif/Supr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/R.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>1051</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* styles.qss */

* {
    font-family: &quot;Playfair Display&quot;, serif; /* Fallback to generic serif font if Playfair Display is not available */
    color: black;
}
/* Style for QStackedWidget */
QStackedWidget {
    background-color: rgba(255, 255, 255, 0.9); /* Slightly transparent white background */
}


/* Style for QGroupBox */


/* Style for QLineEdit (input box) */
QLineEdit {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

/* Style for QComboBox */
QComboBox {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: black;
}

/* Style for QDateEdit */
QDateEdit {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

/* Style for QLabel */
QLabel {
    color: #e67e22; /* Change to a bright orange color */
    font-size: 16px;
    font-weight: bold;
    margin: 5px;
}

/* Style for QPushButton */
QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #d35400; /* Darker orange color on hover */
}

QPushButton:pressed {
    background-color: #a04000; /* Even darker orange color on press */
}

/* Style for QTableView */
QTableView {
    background-color: rgba(255, 255, 255, 0.8); /* Slightly transparent white background */
    border: 5px solid #e67e22; /* Creative thick orange border */
    border-radius: 15px; /* Rounded corners for a more creative look */
    selection-background-color: #e67e22; /* Change to a bright orange color */
}

/* Style for QHeaderView */
QHeaderView::section {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    color: white;
    padding: 5px;
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
}

/* Style for QTableWidgetItem */
QTableView QTableWidget::item {
    background-color: rgba(255, 255, 255, 0.8); /* Slightly transparent white background */
    color: black;
    padding: 5px;
}

QTableView QTableWidget::item:selected {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    color: white;
}

QTableView QTableWidget::item:alternate {
    background-color: #ecf0f1;
    color: black;
}

/* Style for QTabWidget */
QTabWidget {
    background-color: #ffffff; /* Set background color for the tab widget */
}

/* Style for QTabBar */
QTabBar {
    background-color: #e67e22; /* Set the background color for the tab bar */
}

/* Style for selected tab */
QTabBar::tab:selected {
    background-color: #d35400; /* Darker orange color for the selected tab */
    color: white; /* Text color for the selected tab */
    border: 2px solid #a04000; /* Border style for the selected tab */
    border-bottom: none; /* Remove bottom border to create a top border effect */
}

/* Style for unselected tabs */
QTabBar::tab {
    background-color: #e67e22; /* Default background color for tabs */
    color: white; /* Default text color for tabs */
    border: 2px solid #d35400; /* Default border style for tabs */
    border-bottom: none; /* Remove bottom border to create a top border effect */
}

/* Style for unselected tabs on hover */
QTabBar::tab:hover {
    background-color: #d35400; /* Darker orange color on hover for unselected tabs */
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>nom d'oeuvre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>prix</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>discription</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_o">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="date_o">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prix_o">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="discription_o">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_oeuvre">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>431</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>nom artiste</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_a">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="QR">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>510</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>QR</string>
     </property>
    </widget>
    <widget class="QLabel" name="QRCODE">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>310</y>
       <width>401</width>
       <height>311</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_24">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
